test/unit/db/seeds_test.rb

```diff
@@ -3,7 +3,30 @@
 require 'test_helper'
 
 class SeedsTest < ActiveSupport::TestCase
+  class SeedDatabaseConnection < ActiveRecord::Base
+    def self.abstract_class?
+      true
+    end
+  end
+
+  setup do
+    @config = ActiveRecord::Base.configurations['test_seed']
+    adapter = ActiveRecord::Tasks::DatabaseTasks.send(:class_for_adapter, config['adapter']).new(config)
+    creation_options = adapter.send(:creation_options)
```

Comment:
This only works for MySQL.

https://github.com/rails/rails/blob/7f2327f65b4117384bec3547ccd5a99028df57e0/activerecord/lib/active_record/tasks/mysql_database_tasks.rb#L16
https://github.com/rails/rails/blob/7f2327f65b4117384bec3547ccd5a99028df57e0/activerecord/lib/active_record/tasks/postgresql_database_tasks.rb#L15
https://github.com/rsim/oracle-enhanced/blob/2df95002d0352a77a54d075a28fa4dcdabf2b700/lib/active_record/connection_adapters/oracle_enhanced/database_tasks.rb